### PR TITLE
Getting started with Kubernetes

### LINKED ISSUE #1236: Getting Started with Kubernetes
### Brief Summary: 
Kubernetes is an open-source container-orchestration system for automating computer application deployment, scaling, and management. In the article, I will be explaining how various Kubernetes components work and how they are setup.

### Key Takeaways: 
1. What is Kubernetes.
2. How to install Kubernetes on the local computer for development.
3. Various Kubernetes components.
4. Basic Kubernetes commands.

### References: 
N/A

### PR DESCRIPTION
**Pull request by paulodhiambo**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #1236

### CODE REVIEW

**Comment by BrianaNzivu** [comment]
@hectorkambow Hey. 
I wish to review this article. 
Regards.

**Comment by BrianaNzivu** [comment]
Hey, @paulodhiambo . This is a great article and very informative. Kudos!
Just a few changes and you will be good to go.

File: articles/introduction-to-kubernetes/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+Kubernetes is a framework that runs containerized apps across a cluster of machines and coordinates them. It is designed to manage the lifecycle of containers and services providing scalability, predictability, and availability.
```
Comment (hectorkambow) [inline articles/introduction-to-kubernetes/index.md:1]: @BrianaNzivu  

You can leave you comments and suggestions via these in-line comments.

File: articles/introduction-to-kubernetes/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+### Installing Kubernetes locally
+Kubernetes is bundled with [Docker desktop](https://www.docker.com/products/docker-desktop) which is available for both Windows and macOS. Tutorial on how to install docker on Linux can be found [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+To verify if your Kubernetes installation is working, run the below command.
+```bash
+$ kubectl version --client
+```
+
+### Conclusion
```
Comment (BrianaNzivu) [inline articles/introduction-to-kubernetes/index.md:120]: Hey @paulodhiambo. Good job!
Just a few things to bring to your attention:
- Your article has a lot of passive voice statements which is not advisable when it comes to technical writing. Always try using active voice.
- You have quite a number of grammar mistakes. What you can do is before opening a PR you can use some tools such as Grammarly to cross-check your work.
You can access the links of various tools from Section's [resource](https://github.com/section-io/engineering-education/blob/master/new_contributors/resources-page.md) page.

**Review by BrianaNzivu — CHANGES_REQUESTED** [review]

**Review by BrianaNzivu — APPROVED** [review]
Great! @paulodhiambo 
This is ok now.
Proceeding with tech review.

**Comment by BrianaNzivu** [comment]
Finished Tech review.
All is good.
@hectorkambow - Hey, this article needs a plagiarism check and is ready for a final review.

**Comment by BrianaNzivu** [comment]
Hey @paulodhiambo . Please take note of the minor changes I have made.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**